config-store-k8s-config-map/pom.xml

```diff
@@ -0,0 +1,171 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<parent>
+		<artifactId>stargate</artifactId>
+		<groupId>io.stargate</groupId>
+		<version>0.0.18-SNAPSHOT</version>
+	</parent>
+	<modelVersion>4.0.0</modelVersion>
+    <groupId>io.stargate.config.store.k8s.config.map</groupId>
+	<artifactId>config-store-k8s-config-map</artifactId>
+	<dependencies>
+      <dependency>
+        <groupId>io.stargate.config-store</groupId>
+        <artifactId>config-store-api</artifactId>
+        <version>${project.version}</version>
+        <scope>provided</scope>
+      </dependency>
+      <dependency>
+        <groupId>io.fabric8</groupId>
+        <artifactId>kubernetes-client</artifactId>
+        <version>4.12.0</version>
+      </dependency>
```

Comment:
This dependency shouldn't be needed since you're just working with a yaml file.

Reply:
Yes, I am discussing this with Jim Dickinson. The approach that I took here was to use the client to get the config-map, but I will rework it to the YAML approach. To sum up, my finding at this moment:
1. The `extension-a.yaml` needs to be created somehow and provided by the k8s admin at deployment time
2. Then we would need to do smth like kubectl `create configmap from-yaml --from-file=./extension-a.yaml`
3. next, the actual stargate pod will need to mount this in the fs and use it in volumes:
```
      volumeMounts:
      - name: config-volume
        mountPath: /etc/config
  volumes:
    - name: config-volume
      configMap:
        # Provide the name of the ConfigMap containing the files you want
        # to add to the container
        name: from-yaml
```
4. the `extension-a.yaml` is available within to the stragate process.

Reply:
If the end-user wants to reload the file, he can do:
```
kubectl apply -f my-configmap.yaml
emacs my-configmap.yaml
kubectl apply -f my-configmap.yaml
```
it will refresh the file, and the change will be available within the file system (this will be implemented as a part of https://github.com/stargate/stargate/issues/303)